### PR TITLE
add a fileserver for frontend and healthcheck

### PR DESCRIPTION
**Pull request by ssikdar1**
* Add handler to serve the React App.
* add handler for a health check

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -1,59 +1,70 @@
...
-        Handler: client_api_server,
-    }
-
-    //log.Fatal(s.ListenAndServeTLS("cert.pem", "key.pem"))
-    // Debugging purposes
-    log.Fatal(s.ListenAndServe())
+	// React App
+	// TODO if this were a real app: serve with NGINX?
+	path := "/Users/ssikdar1/go/src/github.com/rgdaddio/gobotics/gobotics-frontend/build/"
+	log.Fatal(http.ListenAndServe(":8080", staticFileHandler(http.FileServer(http.Dir(path)))))
```
Comment (rsikdar) [inline server/server.go]: Won't this call block forever? which means the handlers below don't get registered?

File: server/server.go [hunk 2]
```diff
@@ -1,59 +1,70 @@
...
+
+	client_api_server.Handle("/client/die", http.HandlerFunc(die))
+	client_api_server.Handle("/client/device", http.HandlerFunc(device))
+	client_api_server.Handle("/client/devices", http.HandlerFunc(devices))
+	client_api_server.Handle("/health", http.HandlerFunc(healthcheck))
+
+	httpMux := reflect.ValueOf(client_api_server).Elem()
+	finList := httpMux.FieldByIndex([]int{1})
```
Comment (rsikdar) [inline server/server.go]: what are you trying to do here?

File: server/server.go [hunk 3]
```diff
@@ -1,59 +1,70 @@
...
+	"log"
+	"net/http"
+	"os"
+	"reflect"
 )
 
+//TODO add license
+//TODO Godocs?
+//TODO unit tests
 var db *sql.DB // global variable to share it between main and the HTTP handler
```
Comment (rsikdar) [inline server/server.go]: just noticed this now, but I don't think db can be accessed from main unless it's made to be DB

File: server/server.go (hunk 1 above)
Comment (ssikdar1) [reply server/server.go]: 9da924c

**Pull request merged**